src/orchestrator-process/src/lib.rs

```diff
@@ -304,10 +304,17 @@ impl NamespacedOrchestrator for NamespacedProcessOrchestrator {
                     match system.process(pid) {
                         None => (None, None),
                         Some(process) => {
-                            // TODO(benesch): find a way to express this that
-                            // does not involve using `as`.
-                            #[allow(clippy::as_conversions)]
-                            let cpu = (process.cpu_usage() * 10_000_000.0) as u64;
+                            // Justification for `unwrap`:
+                            //
+                            // `u64::try_cast_from(f: f64)`
+                            // will always succeed if 0 <= f <= 2^53.
+                            // Since the max value of `process.cpu_usage()` is
+                            // 100.0 * num_of_cores, this will be true whenever there
+                            // are less than 2^53 / 10^9 logical cores, or about
+                            // 9 million.
+                            let cpu =
+                                u64::try_cast_from(f64::from(process.cpu_usage()) * 10_000_000.0)
+                                    .unwrap();
```

Comment:
Yeah, especially since this code doesn't run in prod. (The process orchestrator is only used in local dev work)

Reply:
oh, whoops, this is completely wrong though, since `try_cast_from` doesn't do rounding.

Will re-think...

Reply:
Sorry, I missed that try_cast_from also does the `as` back to verify the conversion.  What about `f64::trunc` before the cast, since as's behavior for float to int is to round towards zero?

Reply:
yep, that's what I ended up doing.